addons/mass_mailing/static/src/scss/mass_mailing.wysiwyg.scss

```diff
@@ -61,6 +61,21 @@
         transform: translateX(0);
 
         > .o_we_customize_panel .oe-toolbar {
+            grid-template-areas:
+                "typo typo style style colors"
+                "size align list list link"
+                "ai translate translate . ."
+                "options options options options options"
+                "options2 options2 options2 options2 options2"
+                "options3 options3 options3 options3 options3";
+            grid-template-rows:
+                minmax($o-we-sidebar-content-field-height, auto)
+                minmax($o-we-sidebar-content-field-height, auto)
+                minmax($o-we-sidebar-content-field-height, auto)
+                auto
+                auto
+                auto;
+
```

Comment:
Isn't there an scss file somewhere in which this can be put once so that it does not end up being duplicated?

Reply:
I have removed `grid-template-rows` and only kept `grid-template-areas` now. I had to override it because, in the `mass_mailing`, we want to position the translate dropdown next to the AI button, as the animate and highlight buttons are not present in `mass_mailing`.